src/codes.ts

```diff
@@ -8,38 +8,33 @@ class ErrorInvalidCode extends Error {
   }
 }
 
-enum UnspentType {
-  p2sh = 'p2sh',
-  p2shP2wsh = 'p2shP2wsh',
-  p2wsh = 'p2wsh',
-}
+enum EUnspentType { p2sh, p2shP2wsh, p2wsh }
 
-enum Purpose {
-  internal = 'internal',
-  external = 'external',
-}
+const UnspentType = tcomb.enums(EUnspentType, 'UnspentType');
+
+enum EPurpose { internal, external }
```

Comment:
same here, please keep the string values and remove the `E` prefix (In contrast, the `I` prefix for interfaces is common and mandatory due to tslint)

Reply:
Undone.